pkg/daemon/ceph/client/pool.go

```diff
@@ -232,14 +232,30 @@ func givePoolAppTag(context *clusterd.Context, namespace string, poolName string
 }
 
 func setCommonPoolProperties(context *clusterd.Context, pool cephv1.PoolSpec, namespace, poolName, appName string) error {
+	targetSizeRatioProperty := "target_size_ratio"
+	if pool.Replicated.IsTargetRatioEnabled() {
+		err := SetPoolProperty(context, namespace, poolName, targetSizeRatioProperty, strconv.FormatFloat(pool.Replicated.TargetSizeRatio, 'f', -1, 32))
+		if err != nil {
+			return errors.Wrapf(err, "failed to set property %q to replicated pool %q to %.2f", targetSizeRatioProperty, poolName, pool.Replicated.TargetSizeRatio)
+		}
+	}
+
 	compressionModeProperty := "compression_mode"
-	if pool.CompressionMode != "" {
+	if pool.IsCompressionEnabled() {
 		err := SetPoolProperty(context, namespace, poolName, compressionModeProperty, pool.CompressionMode)
 		if err != nil {
 			return errors.Wrapf(err, "failed to set property %q on pool %q to %q", compressionModeProperty, poolName, pool.CompressionMode)
 		}
 	}
 
+	PGAutoscaleModeProperty := "pg_autoscale_mode"
```

Comment:
nit: define this as a const at the top of the source, or else just use the string inline. The same would apply to the target size and compression mode properties above.